zkevm-circuits/src/evm_circuit/execution/memory.rs

```diff
@@ -368,7 +398,10 @@ mod test {
         };
 
         for opcode in [OpcodeId::MSTORE, OpcodeId::MLOAD, OpcodeId::MSTORE8] {
-            let address = rand_word() % (1u64 << 37);
+            // TODO: tracer needs to be optimized to enable larger
+            // max_memory_size_pow_of_two
```

Comment:
-> TODO: tracer needs to be optimized to enable larger max_memory_size_pow_of_two